hyperspy/samfire_utils/samfire_worker.py

```diff
@@ -23,9 +23,6 @@
 from queue import Empty
 import dill
 import numpy as np
-import matplotlib
-
-matplotlib.rcParams['backend'] = 'Agg'
 
```

Comment:
@to266 will know better, but I think that it'll be good to keep things as they were in this case because the worker will certainly never need interactive plotting and, as far as I remember, this was preventing a backend related crash.